antenna_spacing.py

```diff
@@ -0,0 +1,76 @@
+# coding: utf-8
+
+###############################################################
+#  Script     : antenna_spacing.py
+#  Author     : Fanny Ekblom Johansson
+#  Date       : 2017-10-15
+#  Last Edited: 2017-10-18, Henrik Engström
+#  Description: takes a .csv genereted by h5_dumpmeta.py,
+#               keeps the FID-column, adds antenna spaceing
+#               and exports it as a new .txt-file in /offsets
+##############################################################
+# Purpose:
+# - generate input required by join_radar.py
+# Requirements:
+# - irlib, python 2/3 and dumpmeta.csv file as input
+# Notes:
+# - she is an angel
```

Comment:
Would it be appropriate to add FEJ to the contributors list (`CONTRIBUTORS.md`)? I'll let you decide.